### PR TITLE
Update rkt API version to 1.6.0

### LINKED ISSUE #25430: kubectl: exec errors with a segfault when the kubelet's runtime is rkt
I spun up a libvirt coreos+k8s cluster that used rkt for its runtime, and `kubectl exec` would error out with a segfault every time it was used, regardless of if the requested binary existed or not.

```
derek@rokot ~/git/kubernetes> kubectl exec frontend-gbecy ls
error: error executing remote command: error executing command in container: signal: segmentation fault (core dumped)
```

The changes that I made to the cloud configs such that it would be a rktnetes cluster are [here](https://github.com/dgonyeo/kubernetes/commit/6af938e72b9bb93224ec2da75dacad234c081c29).

cc @yifan-gu

### PR DESCRIPTION
**Pull request by tmrts**
Updates coreos/rkt container runtime dependency to `1.6.0`

Fixes #25430

### CODE REVIEW

**Comment by tmrts** [comment]
cc @yifan-gu

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
 	"ImportPath": "k8s.io/kubernetes",
 	"GoVersion": "go1.6",
-	"GodepVersion": "v63",
+	"GodepVersion": "v67",
```
Comment (yifan-gu) [inline Godeps/Godeps.json]: @tmrts Can you revert the change?

**Comment by yifan-gu** [comment]
@tmrts Can we also change these defaults here ?https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/rkt/rkt.go#L65-L67

**Comment by tmrts** [comment]
@yifan-gu: on it :+1:

**Comment by tmrts** [comment]
@yifan-gu PTAL

File: pkg/kubelet/rkt/rkt.go [hunk 2]
```diff
@@ -62,11 +62,11 @@ const (
...
 
-	minimumAppcVersion       = "0.7.4"
-	minimumRktBinVersion     = "1.2.1"
-	recommendedRktBinVersion = "1.2.1"
+	minimumAppcVersion       = "0.8.1"
+	minimumRktBinVersion     = "1.6.0"
+	recommendedRktBinVersion = "1.6.0"
 	minimumRktApiVersion     = "1.0.0-alpha"
-	minimumSystemdVersion    = "219"
+	minimumSystemdVersion    = "229"
```
Comment (yifan-gu) [inline pkg/kubelet/rkt/rkt.go]: @tmrts Is this required? I think this is the systemd on the host, so 219 is ok. cc @sjpotter

Comment (tmrts) [inline pkg/kubelet/rkt/rkt.go]: @yifan-gu I believe you're right. I'll change it back

File: pkg/kubelet/rkt/rkt.go [hunk 3]
```diff
@@ -62,9 +62,9 @@ const (
 	RktType                      = "rkt"
 	DefaultRktAPIServiceEndpoint = "localhost:15441"
 
-	minimumAppcVersion       = "0.7.4"
-	minimumRktBinVersion     = "1.2.1"
-	recommendedRktBinVersion = "1.2.1"
+	minimumAppcVersion       = "0.8.1"
+	minimumRktBinVersion     = "1.6.0"
```
Comment (yifan-gu) [inline pkg/kubelet/rkt/rkt.go]: @euank Just heads up, we will need to switch to 1.6 in the testing infra :)

**Comment by yifan-gu** [comment]
LGTM

**Comment by euank** [comment]
Fixes #25430

**Comment by jonboulle** [comment]
lgtm

File: pkg/kubelet/rkt/rkt.go [hunk 4]
```diff
@@ -62,9 +62,9 @@ const (
 	RktType                      = "rkt"
 	DefaultRktAPIServiceEndpoint = "localhost:15441"
 
-	minimumAppcVersion       = "0.7.4"
-	minimumRktBinVersion     = "1.2.1"
-	recommendedRktBinVersion = "1.2.1"
+	minimumAppcVersion       = "0.8.1"
```
Comment (euank) [inline pkg/kubelet/rkt/rkt.go]: I don't think we should need appc version since it should be strongly tied to rkt version, right?

Comment (yifan-gu) [inline pkg/kubelet/rkt/rkt.go]: @euank I think so. (Actually the rkt api version is not required as well, as it's always binded to rkt release version) cc @jonboulle 
Let's address this in a follow up PR.
